edweiss-firebase/functions/src/functions/lectures/createQuestion.ts

```diff
@@ -1,10 +1,16 @@
 import { Timestamp } from 'firebase-admin/firestore';
 import LectureDisplay from 'model/lectures/lectureDoc';
 import { onAuthentifiedCall } from 'utils/firebase';
-import { addDocument, CollectionOf } from 'utils/firestore';
+import { addDocument, CollectionOf, Collections, getRequiredDocument } from 'utils/firestore';
 import { fail, ok } from 'utils/status';
 
+
 export const createQuestion = onAuthentifiedCall(LectureDisplay.Functions.createQuestion, async (userId, args) => {
```

Comment:
Same as before, you might want to use `onSanitizedCall` to ensure full safety here. This function is vulnerable in its current state.